content/overview/technology/appropriate-deployments.md

```diff
@@ -11,4 +11,4 @@ aliases:
 
 We use the term “service” or “application” when talking about what gets deployed on cloud.gov because it is absolutely possible to deploy “non-website” services using it.
 
-As we work with agency partners on bigger and more complex delivery projects, we will regularly evaluate whether cloud.gov is a suitable choice, and react accordingly if we find ourselves working on a system that doesn’t map easily to the PaaS model.
+As we work with agency partners on bigger and more complex delivery projects, we will regularly evaluate whether cloud.gov is a suitable choice. If we decide we are working on a system that doesn’t map easily to the PaaS model, we will react accordingly.
```

Comment:
This is a confusing paragraph that we can jettison. Maybe we should move the first paragraph to a different page and delete this page?